tests/test-app/tests/apollo-cursor-pagination/knex-offset-implementation.test.js

```diff
@@ -0,0 +1,361 @@
+const _ = require('../helpers/database'); // eslint-disable-line no-unused-vars
+const { start, stop, graphqlQuery } = require('../helpers/integration-server');
+const catFactory = require('../factories/mocks/cat');
+
+describe('getCatsByOwner root query', () => {
```

Comment:
do you have the image of the tests passing?

Reply:
Added in the description